frontend/src/scenes/surveys/SurveyEdit.tsx

```diff
@@ -595,6 +596,37 @@ export default function SurveyEdit(): JSX.Element {
                                                     )}
                                                 </BindLogic>
                                             </LemonField.Pure>
+                                            {featureFlags[FEATURE_FLAGS.SURVEYS_EVENTS] && (
+                                                <LemonField.Pure label="User sends events">
+                                                    <EventSelect
+                                                        onChange={(includedEvents) => {
+                                                            setSurveyValue('conditions', {
+                                                                events: {
```

Comment:
this will remove any other existing conditions which isn't great 😅, should append to conditions

Reply:
good eye! fixed now